### ISSUE TITLE
Ignore ads if not in radius

### ISSUE THREAD

**Issue by nick-traeger** [DanielStefanK/kleinanzeigen-alert#1]
Hello,
just using the bot you are hosting on telegram and found a small bug. If no result was found, then "Alternative Anzeigen in der Umgebung" are output, but which do not belong in the search radius. The bot then outputs those as well, but it would be nice if it ignored those. Here is an example of the search:

https://www.ebay-kleinanzeigen.de/s-27612/palettenrahmen/k0l2703

**Comment by DanielStefanK** [comment]
So you entered a query like `/add palettenrahmen, 27612, 5` and it returned ads that are outside of the 5 km radius?
I tried to reproduce this behaviour, but for the given query I don't get any ads (which is correct).
Do you have an example of the query not working? Do you have any other queries that send you the unwanted ads ?

**Comment by nick-traeger** [comment]
That is correct, I have also attached screenshots. Other search terms are not affected, but they always had at least one result.

![Screenshot_20210625_135350](https://user-images.githubusercontent.com/11520139/123421593-536b2c80-d5bd-11eb-9895-2a15218addd4.jpg)
![Screenshot_20210625_135250](https://user-images.githubusercontent.com/11520139/123421597-5403c300-d5bd-11eb-9dc6-197bf7bf4c62.jpg)

**Comment by DanielStefanK** [comment]
I think I reproduced the bug.
Will look for a fix over the weekend.

**Comment by DanielStefanK** [comment]
The fixed version is deployed

**Comment by nick-traeger** [comment]
Thank you! Your bot is very helpful
